### PR TITLE
Shared Word Card Page Visual Improvement

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/wordnote/issues/165

## TODOs
- [x] Make sure it says width 100%, until it reaches 1000px. after 1000px, it shows only the 1000px.

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `Project` is created & linked, if multiple PRs are associated to this PR
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

**Review by mlajkim — COMMENTED** [review]
You sure this works?

**Comment by mlajkim** [comment]
reference: https://github.com/ajktown/wordnote/compare/3b715311f6128f9195970e7f458ab12cc4538914...87a466e

TODO:
- [ ] Delete branch [shared-word-card-page-visual-improvement-next-movement-sample](https://github.com/ajktown/wordnote/tree/shared-word-card-page-visual-improvement-next-movement-sample) after the reference deleted

File: src/components/molecule_word_card/index.shared.tsx [hunk 1]
```diff
@@ -41,7 +43,13 @@ const WordCardShared: FC<Props> = ({ wordId }) => {
...
   return (
     <StyledSuspense>
-      <Card style={{ width: `100%`, borderRadius: 9 }}>
+      <Card
+        style={{
+          width: width > 1000 ? `1000px` : `100%`,
+          maxWidth: `1000px`,
+          borderRadius: 9,
+        }}
+      >
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.shared.tsx:52]: dont forget to delete

File: src/components/molecule_word_card/index.shared.tsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { PageQueryConst } from '@/constants/page-queries.constant'
 import StyledCountdownTimer from '@/atoms/StyledCountdownTimer'
 import TagButtonLanguage from '../atom_tag_chip/index.language'
 import TagChipCustomized from '../atom_tag_chip/index.customized'
+import { useWindowSize } from 'react-use'
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.shared.tsx:15]: dont forget to delete

File: src/components/molecule_word_card/index.shared.tsx [hunk 3]
```diff
@@ -20,6 +21,7 @@ interface Props {
 const URL_PATH = `/` + PageConst.Share + `?` + PageQueryConst.wordID + `=`
 
 const WordCardShared: FC<Props> = ({ wordId }) => {
+  const { width } = useWindowSize()
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.shared.tsx:24]: dont forget to delete

**Review by mlajkim — COMMENTED** [review]
second review

File: src/components/molecule_word_card/index.shared.tsx (hunk 1 above)
Comment (akanex115) [reply src/components/molecule_word_card/index.shared.tsx]: Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)

File: src/components/molecule_word_card/index.shared.tsx (hunk 2 above)
Comment (akanex115) [reply src/components/molecule_word_card/index.shared.tsx]: Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)

File: src/components/molecule_word_card/index.shared.tsx (hunk 3 above)
Comment (akanex115) [reply src/components/molecule_word_card/index.shared.tsx]: Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)

File: src/components/organism_shared_word_card_frame/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <Typography variant="h5">
+          {`Beta feature: Below is the shared word card`}
+        </Typography>
+        <Stack>
+          <WordCardShared wordId={wordId.trim()} />
+        </Stack>
+        <Box mb={1} />
+        <SharedWordCardAddWordButtonPart wordId={wordId.trim()} />
+      </Stack>
+    </StyledCentered>
```
[the remark this answers is not in the document]
Comment (akanex115) [reply src/components/organism_shared_word_card_frame/index.tsx]: Handled in [41e18eb](https://github.com/ajktown/wordnote/pull/205/commits/41e18eb3b897145e6d9f02c2827aa167ade05ddb)

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**